src/components/TodoContainer.js

```diff
@@ -0,0 +1,97 @@
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable no-param-reassign */
+import React, { Component } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import TodosList from './TodosList';
+import Header from './Header';
+import InputTodo from './InputTodo';
+
+class TodoContainer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      todos: [],
+    };
+  }
+
+  componentDidMount() {
+    const temp = localStorage.getItem('todos');
+    const loadedTodos = JSON.parse(temp);
+    if (loadedTodos) {
+      this.setState({
+        todos: loadedTodos,
+      });
+    }
+  }
```

Comment:
Nice job using lifecycle events and localStorage.

Reply:
Thank you for your feedback I appreciate that 👏🏼